<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineedit_Nume">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nume</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Prenume">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Prenume</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Parola">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Parola</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_admin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_doctor">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Doctor</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_pacient">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Pacient</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>popoasd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mmnk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sda</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>asdas</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bucureasa</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
